### PR TITLE
Feature/docs to slash validator

### PR DESCRIPTION
**Pull request by cducrest**
closes https://github.com/trustlines-network/project/issues/504?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,45 +73,56 @@ docker run --name monitor -d --restart=always --link trustlines-testnet:parity -
...
 
 When the monitor reported an equivocation by a malicious validator, it is
-possible to remove that validator from the validator set. The information to proof the
-equivocation must be provided via the `reportMaliciousValidator` function to the
-validator set contract. This information can be found at the created report
-file, located at the directory defined with `--report-dir`. The file is named
-like `equivocation_reports_for_proposer_0x...` followed by the address of the
+possible to remove that validator from the validator set and slash their deposit on the Ethereum main chain.
+The information required to prove the equivocation can be found in the created report file
+located at the directory defined with `--report-dir` (default `/reports`). The file is named
```
Comment (weilbith) [inline README.md]:
```suggestion
located at the directory defined with `--report-dir` (default `./reports`). The file is named
```

This is not an absolute path.

File: README.md [hunk 2]
```diff
@@ -73,45 +73,56 @@ docker run --name monitor -d --restart=always --link trustlines-testnet:parity -
...
 
-Please make sure to use the currently active validator set contract address,
-which can be found within the [chain
-specification](https://github.com/trustlines-protocol/blockchain/blob/836e456d5ed8bcb576986e1c4cfe60603d14dcd0/config/trustlines-spec.json#L7).
-
-The former example assumes a running _Parity_ node on `http://localhost:8545`
-with an account unlocked to sign the transaction. The URL of the parity node can
-be specified with the `--jsonrpc` option. `--keystore` can be specified to sign
-the transaction with a local key instead of relying on an unlocked account.
+In the above example, `/reports` is the directory containing the reports of equivocating
```
Comment (weilbith) [inline README.md]:
```suggestion
In the above example, `./reports` is the directory containing the reports of equivocating
```

File: README.md [hunk 3]
```diff
@@ -73,45 +73,56 @@ docker run --name monitor -d --restart=always --link trustlines-testnet:parity -
...
+validator with address `0x505ab22ef8f3ae874dec92e60665ca490fb68192`,
+the address of the reporting contract is `0x9Cc30A6088DB80F8a3B2b4d2f491AbC98559C59c`.
+It assumes a running _Parity_ node on `http://127.0.0.1:8545` syncing the desired chain:
+Ethereum main chain for slashing and Trustlines chain for removing a validator.
```
Comment (weilbith) [inline README.md]:
```suggestion
Ethereum main chain for slashing or Trustlines chain for removing a validator.
```

Comment (cducrest) [reply README.md]: Ok

File: README.md (hunk 2 above)
Comment (cducrest) [reply README.md]: Ok

File: README.md (hunk 1 above)
Comment (cducrest) [reply README.md]: Ok

**Pull request merged**